src/script/components/calling/GroupVideoGrid.tsx

```diff
@@ -85,7 +84,7 @@ const GroupVideoGrid: React.FunctionComponent<GroupVideoGripProps> = ({
     if (grid.thumbnail) {
       return;
     }
-    const participant = videoParticipants.find(participant => participant.doesMatchIds(userId, clientId));
+    const participant = grid.grid.find(participant => participant?.doesMatchIds(userId, clientId));
```

Comment:
`grid.grid`?

Reply:
Yes, the naming is a bit unfortunate ...